server/routes/referrals/referralFormPresenter.ts

```diff
@@ -47,7 +47,12 @@ export default class ReferralFormPresenter {
         },
         {
           title: 'Select desired outcomes',
-          url: this.calculateTaskUrl('desired-outcomes', this.taskValues.relevantSentence),
+          url: this.calculateTaskUrl(
+            this.referral.serviceCategoryIds
+              ? `service-category/${this.referral.serviceCategoryIds[0]}/desired-outcomes`
+              : null,
```

Comment:
i didn't see a test for this case, what would the behaviour be if we hit this?

Reply:
Have added this in, there's a bit more detail that will come in the test once we add the cohort referral journey, but this is at least covered for now.